ansible/library/minigraph_facts.py

```diff
@@ -637,12 +641,19 @@ def parse_xml(filename, hostname, asic_name=None):
     global port_alias_to_name_map
     global port_name_to_alias_map
     global port_alias_asic_map
+    global port_alias_to_port_asic_alias_map
+    global port_name_to_index_map
 
-    port_alias_to_name_map, port_alias_asic_map = get_port_alias_to_name_map(hwsku, asic_id)
+    port_alias_to_name_map, port_alias_asic_map, port_name_to_index_map = get_port_alias_to_name_map(hwsku, asic_name)
 
     # Create inverse mapping between port name and alias
     port_name_to_alias_map = {v: k for k, v in port_alias_to_name_map.items()}
 
+    for k, v in port_alias_to_name_map.items():
+        for i, j in port_alias_asic_map.items():
+            if v == j:
+                port_alias_to_port_asic_alias_map[k] = i
```

Comment:
Could we add a comment explaining the key and value of the new map added.

Reply:
@SuvarnaMeenakshi I will take care of other PR.